modules/drivers/oracle/src/metabase/driver/oracle.clj

```diff
@@ -181,6 +181,10 @@
   [driver _ v]
   (sql.qp/adjust-start-of-week driver (partial trunc :day) v))
 
+(defmethod sql.qp/date [:oracle :week-of-year-iso]
+  [_ _ v]
+  (hx/->integer (hsql/call :to_char v (hx/literal :iw))))
```

Comment:
what's the point of the `to_char` call here?
edit: the `iw` arg is
> Week of year (1-52 or 1-53) based on the ISO standard.
https://docs.oracle.com/cd/B19306_01/server.102/b14200/sql_elements004.htm#i34510
Might warrant a comment or a link to that doc